app/lib/search/backend.dart

```diff
@@ -135,7 +138,7 @@ class SearchBackend {
       description: compactDescription(pv.pubspec.description),
       created: p.created,
       updated: pv.created,
-      readme: compactReadme(pv.readmeContent),
+      readme: compactReadme(readmeAsset?.textContent),
```

Comment:
Should we bump runtime version? or something else that'll cause a new version of the search index and snapshot?

Reply:
It is already bumped because of other changes, but it shouldn't matter: the values must be the same.